benchmarks/kernels/benchmark_moe.py

```diff
@@ -155,11 +177,16 @@ def benchmark(
         hidden_size: int,
         topk: int,
         dtype: torch.dtype,
-        use_fp8: bool,
+        use_fp8_w8a8: bool,
+        use_int8_w8a16: bool,
     ) -> Tuple[Dict[str, int], float]:
         torch.cuda.manual_seed_all(self.seed)
 
-        dtype_str = "float8" if use_fp8 else None
+        dtype_str = None
+        if use_fp8_w8a8:
+            dtype_str = "float8"
+        elif use_int8_w8a16:
+            dtype_str = "int8"
```

Comment:
Should these strings be make explicit as well, or are they used as torch dtypes?

Reply:
They are used to load the moe config files, changed it to use the new function that encapsulates this logic in one place